api/v1beta2/foundationdb_env_variables.go

```diff
@@ -21,6 +21,75 @@
 package v1beta2
 
 const (
-	// EnvNamePublicIP defines the FDB_PUBLIC_IP environment variable name.
+	// EnvNameDNSName specifies the DNS locality (identifies the pod when using DNS in lieu of IP)
+	EnvNameDNSName = "FDB_DNS_NAME"
+
+	// EnvNameMachineID specifies the Machine ID locality. Also defines fault domain along with EnvNameZoneID
```

Comment:
```suggestion
	// EnvNameMachineID specifies the Machine ID locality.
```

The latter is not true, in most cases a single fault domain contains more than one machine.